Content.Shared/Movement/Systems/SharedMoverController.cs

```diff
@@ -443,6 +445,14 @@ private bool IsAroundCollider(SharedPhysicsSystem broadPhaseSystem, TransformCom
                 sound = moverModifier.FootstepSoundCollection;
                 return true;
             }
+            
+            // If soft paws and no shoes, no sound. Delta V
+            if (_entities.TryGetComponent(uid, out SoftPawsComponent? _) &
```

Comment:
I would rename this component just because it could be reused for any number of other things